editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -5247,7 +5248,8 @@ void Node3DEditor::_init_indicators() {
 			gizmo_color[i] = mat;
 
 			Ref<StandardMaterial3D> mat_hl = mat->duplicate();
-			mat_hl->set_albedo(Color(col.r * 1.3, col.g * 1.3, col.b * 1.3, 1.0));
+			const float brightness = 1.3;
+			mat_hl->set_albedo(Color(col.r * brightness, col.g * brightness, col.b * brightness, 1.0));
```

Comment:
```suggestion
			mat_hl->set_albedo(Color(col.r * brightness, col.g * brightness, col.b * brightness));
```

As you said: The alpha component of a Color also defaults to `1.0`, so we don't have to specify it.